docs/zh-CN/docs/sql-manual/sql-reference/Account-Management-Statements/LDAP.md

```diff
@@ -44,12 +44,12 @@ SET LDAP_ADMIN_PASSWORD
 
 1. 设置LDAP管理员密码
 ```sql
-SET LDAP_ADMIN_PASSWORD = '123456'
+SET LDAP_ADMIN_PASSWORD = PASSWORD('123456')
 ```
 
 ### Keywords
 
-    LDAP
```

Comment:
how about use LDAP,ADMIN,PASSWORD ?

Reply:
maybe `LDAP, PASSWORD, LDAP_ADMIN_PASSWORD` is good